src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_commands.py

```diff
@@ -37,6 +40,22 @@
 # from .validators import db_up_namespace_processor
 
 
+register_logic_breaking_change('postgres flexible-server create', 'Update default value of "--sku-name"',
+                               detail='The default value will be changed from "Standard_D2s_v3" to a '
+                               'supported sku based on regional capabilities.')
+register_default_value_breaking_change('postgres flexible-server create', '--version', '16', '17')
+register_default_value_breaking_change('postgres flexible-server create', '--create-default-database', 'Enabled',
+                                       'Disabled')
+register_argument_deprecate('postgres flexible-server create', '--active-directory-auth', '--microsoft-entra-auth')
+register_argument_deprecate('postgres flexible-server update', '--active-directory-auth', '--microsoft-entra-auth')
+register_command_group_deprecate('postgres flexible-server ad-admin', redirect='microsoft-entra-admin')
+register_command_deprecate('postgres flexible-server replica stop-replication',
+                           redirect='postgres flexible-server replica promote', hide=True)
+register_other_breaking_change('postgres flexible-server update',
+                               message='User confirmation will be needed for compute and storage updates '
+                               'that trigger a restart of the VM.')
```

Comment:
"that trigger a restart of the VM" => "that trigger a restart of the server". Because, in reality, the server Restart operation doesn't trigger a restart of the VM, but only of the postgres container running on the VM.